dockerfiles/haskell/3.0.0.0/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:22.04
+
+# Install necessary packages for GHC (compiler)
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    ghc cabal-install \
```

Comment:
Can we pin ghc and cabal to specific versions?

Reply:
I managed to fix them, but the build process is significantly longer. You can find the images at https://hub.docker.com/repository/docker/ion606/haskell/general if you want  to manually compare